lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -9,35 +9,37 @@
         </div>
     </div>
   <% if (progress) { %>  
-    <p id="status-<%- id %>" class="certificate-status"><%= interpolate(
-        gettext('You have earned certificates in %(completed_courses)s of the %(total_courses)s courses so far.'),
-        {completed_courses: progress.total.completed, total_courses: progress.total.courses}, true
-    ) %></p>
-  <% } %>
-</div>
-<% if (progress) { %>
-<div class="progress">
-    <div class="bar complete" style="width:<%- progress.percentage.completed %>;"></div>
-    <div class="bar in-progress" style="width:<%- progress.percentage.in_progress %>;">
-        <span class="sr"><%= interpolate(
+    <p class="certificate-status">
+        <a href="<%- marketingUrl %>" class="link secondary"><%= interpolate(
             ngettext(
-                '%(count)s course are in progress.',
-                '%(count)s courses are in progress.',
-                progress.total.in_progress
-            ),
-            {count: progress.total.in_progress}, true 
-        ) %></span>
-    </div>
-    <div class="bar not-started">
-        <span class="sr"><%= interpolate(
-            ngettext(
-                '%(count)s course have not been started.',
+                '%(count)s course has not been started.',
                 '%(count)s courses have not been started.',
                 progress.total.not_started
             ),
             {count: progress.total.not_started}, true 
+        ) %></a>
+
+        <a href="<%- marketingUrl %>" class="link secondary"><%= interpolate(
+            ngettext(
+                '%(count)s course is in progress.',
+                '%(count)s courses are in progress.',
+                progress.total.in_progress
+            ),
+            {count: progress.total.in_progress}, true 
+        ) %></a>
+
+        <span id="status-<%- id %>" class="link"><%= interpolate(
```

Comment:
Good point, it was a link until at the end when I realized that it shouldn't be. I will update the class name to be more meaningful.